test/index.js

```diff
@@ -28,4 +28,10 @@ describe('LiveRegion', function () {
     });
   });
 
+  describe('destroy', function () {
+    it('should remove region node from the DOM', function (done) {
```

Comment:
this `done` parameter is only needed for asynchronous tests (so its neither used nor needed here). I'd also suggest something a little more clear as a check if the element has been removed from the dom. Something like `assert.false(document.contains(polite.region);`

Reply:
Oops, I copied and pasted from the one above 😇. And way better check! I was curious the best way to test that. Love using `contains` 🤘.